src/utils_request.js

```diff
@@ -143,6 +148,10 @@ export const requestAsBrowser = async (options) => {
     }
 };
 
+function getProxyUrl(proxyUrl, proxyConfiguration) {
+    return proxyConfiguration ? proxyConfiguration.getUrl() : proxyUrl;
+}
+
```

Comment:
Why not use the regular expression directly?

Reply:
Looks better, changed :(.